bin/scripts/org_migrator/org_migrator.py

```diff
@@ -455,7 +455,7 @@ def do_export(self):
 
         content_uuid_query1 = 'SELECT oc.content_uuid FROM cp2_owner_content oc WHERE oc.owner_id = %s' + \
             'UNION ' + \
-            'SELECT ec.content_uuid FROM cp2_environment_content ec JOIN cp_environment e on e.id = ec.environment_id WHERE e.owner_id = %s'
+            'SELECT ec.content_id FROM cp_environment_content ec JOIN cp_environment e on e.id = ec.environment_id WHERE e.owner_id = %s'
```

Comment:
This chunk here is for pulling all content UUIDs to ensure we have all the referenced content. With the env content no longer being a hard ref to a specific content instance, we no longer need this query here.